src/GameClient.cpp

```diff
@@ -1684,9 +1684,16 @@ unsigned GameClient::SaveToFile(const std::string& filename)
 
     // Und alles speichern
     if(!save.Save(filename))
+    {
+        GameMessage_System_Chat saveAnnouncement = GameMessage_System_Chat(playerId_, "Error occurred while saving game.");
```

Comment:
Not a good idea: Saving is done per client. You would most likely flood the game with messages from every player. Rather just show it to the local player maybe using the notification system.
Also use the localization feature: `_("Translated text")`